flash/template/classification/model.py

```diff
@@ -67,7 +67,7 @@ def __init__(
             metrics=metrics,
             learning_rate=learning_rate,
             multi_label=multi_label,
-            serializer=serializer or Labels(),
+            output=output or Labels(),
```

Comment:
Personal preference, but I would prefer `LabelsOutput` as we would have `InputImagePaths`. This would make it more consistent and readable too.

Reply:
I think we should leave that for a potential follow up as we would need to deprecate `Labels` etc. rather than just rename as they are a public API.